src/main/java/frc/robot/subsystems/vision/PhotonSubsystem.java

```diff
@@ -25,6 +25,9 @@
 import org.photonvision.targeting.PhotonTrackedTarget;
 
 public class PhotonSubsystem extends SubsystemBase implements VisionSubsystem {
+
+  private final boolean TURRET_CAMERA_ENABLED = false;
```

Comment:
Camera is mounted, let's set this to True

Reply:
I was waiting until the second camera was physically mounted because the current camera is the "main" camera.